projects/elife.yaml

```diff
@@ -988,21 +988,21 @@ journal-cms:
             rds:
                 type: db.t2.small
                 engine: MySQL
-                version: '5.7'
-                storage: 24 # GB
+                version: '5.7.23'
+                storage: 48 # GB
                 backup-retention: 2 # days
             ext:
-                size: 30 # GB
+                size: 60 # GB
         prod:
             description: production environment. backed by RDS
             rds:
                 type: db.t2.medium
                 engine: MySQL
-                version: '5.7'
+                version: '5.7.23'
                 multi-az: true
-                storage: 24 # GB
+                storage: 48 # GB
             ext:
-                size: 30 # GB
+                size: 60 # GB
```

Comment:
accidentally increased this as well, and now can't go back

Reply:
that's fine.

if the database is too large for 30GB it has problems regardless.